hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebServicesNodeLabels.java

```diff
@@ -119,527 +132,441 @@ public TestRMWebServicesNodeLabels() {
-            .path("get-node-labels").queryParam("user.name", userName)
+            .path("label-mappings").queryParam("user.name", userName)
             .accept(MediaType.APPLICATION_JSON).get(ClientResponse.class);
+    return response;
+  }
+
+  private ClientResponse getNodeLabelMappingsByLabels(WebResource r, String... labelNames) {
+    MultivaluedMapImpl params = createMultiValuedMap(labelNames);
+    return r.path("ws").path("v1").path("cluster")
+        .path("label-mappings").queryParam("user.name", userName)
+        .queryParams(params)
+        .accept(MediaType.APPLICATION_JSON).get(ClientResponse.class);
+  }
+
+  private ClientResponse replaceLabelsOnNode(WebResource r, String node, String... labelNames) {
+    return replaceLabelsOnNodeWithUserName(r, node, userName, labelNames);
+  }
+
+  private ClientResponse replaceLabelsOnNodeWithUserName(WebResource r, String node,
+      String userName, String... labelNames) {
+    LOG.info("Replacing labels on node '{}', label(s): {}", node, labelNames);
+    MultivaluedMapImpl params = createMultiValuedMap(labelNames);
+    return r.path("ws").path("v1").path("cluster")
+        .path("nodes").path(node)
+        .path("replace-labels")
+        .queryParam("user.name", userName)
+        .queryParams(params)
+        .accept(MediaType.APPLICATION_JSON)
+        .post(ClientResponse.class);
+  }
+
+  private MultivaluedMapImpl createMultiValuedMap(String[] labelNames) {
+    MultivaluedMapImpl params = new MultivaluedMapImpl();
+    for (String labelName : labelNames) {
+      params.add("labels", labelName);
+    }
+    return params;
+  }
+
+  private ClientResponse removeNodeLabel(WebResource r, String... labelNames) {
+    MultivaluedMapImpl params = createMultiValuedMap(labelNames);
+    return r.path("ws").path("v1").path("cluster")
+        .path("remove-node-labels")
+        .queryParam("user.name", userName)
+        .queryParams(params)
+        .accept(MediaType.APPLICATION_JSON)
+        .post(ClientResponse.class);
+  }
+
+  private ClientResponse getLabelsOfNode(WebResource r, String node) {
+    return r.path("ws").path("v1").path("cluster")
+        .path("nodes").path(node)
+        .path("get-labels").queryParam("user.name", userName)
+        .accept(MediaType.APPLICATION_JSON).get(ClientResponse.class);
+  }
+
+  private ClientResponse getNodeLabels(WebResource r) {
+    return r.path("ws").path("v1").path("cluster")
+        .path("get-node-labels")
+        .queryParam("user.name", userName)
+        .accept(MediaType.APPLICATION_JSON).get(ClientResponse.class);
+  }
+
+  private ClientResponse getNodeToLabels(WebResource r) {
+    return r.path("ws").path("v1").path("cluster")
+        .path("get-node-to-labels")
+        .queryParam("user.name", userName)
+        .accept(MediaType.APPLICATION_JSON).get(ClientResponse.class);
+  }
+
+  private ClientResponse addNodeLabels(WebResource r, List<Pair<String, Boolean>> nlInfos) throws Exception {
```

Comment:
./hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebServicesNodeLabels.java:495:  private ClientResponse addNodeLabels(WebResource r, List<Pair<String, Boolean>> nlInfos) throws Exception {: Line is longer than 100 characters (found 109). [LineLength]

It would be awesome if checkstyle could comment in PRs.

Reply:
Yep, would be good.
Fixed the line length issue.